qnm/angular.py

```diff
@@ -118,7 +157,28 @@ def l_min(s, m):
     return np.max([np.abs(s), np.abs(m)])
 
 def M_matrix(s, c, m, l_max):
-    """ TODO Document """
+    """  Symmetric pentadiagonal matrix of size N x N for 
+      N = l_max - l_min +1
+
+    Parameters
+    ----------
+    s: integer
+      Spin-weight of interest
+
+    c: integer
+      Oblateness of the spheroidal harmonic
+
+    m: integer
+      Magnetic quantum number
+
+    l_max: integer
+      Maximum angular quantum number
+
+    Returns
+    -------
+    array
```

Comment:
I think the standard naming is `ndarray`, check https://numpydoc.readthedocs.io/en/latest/format.html